sdk-lib/src/main/java/cash/z/ecc/android/sdk/db/DatabaseCoordinator.kt

```diff
@@ -68,15 +69,36 @@ internal class DatabaseCoordinator private constructor(context: Context) {
     }
 
     /**
-     * Returns the file of the Cache database that would correspond to the given alias
-     * and network attributes.
+     * Returns the root folder of the Cache database that would correspond to the given
+     * alias and network attributes.
+     *
+     * @param network the network associated with the data in the cache database.
+     * @param alias the alias to convert into a database path
+     *
+     * @return the Cache database folder
+     */
+    internal suspend fun cacheDbRoot(
+        network: ZcashNetwork,
+        alias: String
+    ): File {
+        return newDatabaseFilePointer(
+            network,
+            alias,
+            DB_FS_CACHE_NAME,
+            Files.getZcashNoBackupSubdirectory(applicationContext)
+        )
+    }
+
+    /**
+     * Returns the file of the legacy Cache database that would correspond to the given
+     * alias and network attributes.
      *
      * @param network the network associated with the data in the cache database.
      * @param alias the alias to convert into a database path
      *
      * @return the Cache database file
      */
-    internal suspend fun cacheDbFile(
+    internal suspend fun legacyCacheDbFile(
```

Comment:
Kept this function around as I expect we'll want to use it to delete the old caches.